client/experimentalclient/backup.go

```diff
@@ -97,3 +101,37 @@ func (b backupClient) Exist(ctx context.Context, v string) (bool, error) {
 	}
 	return false, fmt.Errorf("check backup existence (%s) failed: unexpected status code (%v), response (%s)", b.addr, resp.Status, errmsg)
 }
+
+func (b backupClient) ServiceStatus(ctx context.Context) (*backupapi.ServiceStatus, error) {
+	req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("%s://%s/status", b.scheme, path.Join(b.addr, backupapi.APIV1)), nil)
+	if err != nil {
+		return nil, fmt.Errorf("get service status (%s) failed: %v", b.addr, err)
+	}
+	req.WithContext(ctx)
```

Comment:
Should be
```
req = req.WithContext(ctx) 
```
Right?
This also needs to be applied to other existing WithContext()?